resources/assets/less/app.less

```diff
@@ -384,6 +384,23 @@ a.logo.no-hover a:hover {
   background-color: transparent;
 }
 
+.help-block {
```

Comment:
Per my previous comments, we use `help-block` in a lot of other places, and we might not want to affect those things. I'd probably create a new class using these attributes and add that class to the boxes. 

TBH, I'm not even sure that this is the right solve here, but it can be tricky to try to handle a grid when the display could be on a iPhone or a 30 inch monitor. :-/